CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+## [15.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v14.5.0...v15.0.0) (2019-08-23)
+
+This release introduces some **breaking changes** related to minimal dependency versions.
```

Comment:
does this mean we need to use `react`, `react-dom` version 19?

Reply:
Yes, we should use React 16.9